content/en/docs/configuration/acme/http01/_index.md

```diff
@@ -155,3 +158,71 @@ The added labels and annotations will merge on top of the cert-manager defaults,
 overriding entries with the same key.
 
 No other fields of the ingress can be edited.
+
+## Configuring the HTTP01 Gateway API solver
+
+This feature, introduced in cert-manager 1.5, is currently experimental and
+subject to change.
+
+{{% pageinfo color="info" %}}
+
+📌  This feature requires the installation of the Gateway API CRDs. No
+additional flag is needed on the cert-manager controller; cert-manager discovers
```

Comment:
I like this info box. Shall we also use an info box in the gateway shim docs, for consistency.

Reply:
Done!